lua/luasnip/extras/snippet_list.lua

```diff
@@ -0,0 +1,92 @@
+local function get_context(snippet)
+	return {
+		name = snippet.name,
+		trigger = snippet.trigger,
+		description = snippet.dscr,
+		wordTrig = snippet.wordTrig and true or false,
+		regTrig = snippet.regTrig and true or false,
+		docstring = snippet:get_docstring(),
+	}
+end
+
+local function get_name(bufnr)
+	return "LuaSnip://Snippets buf " .. bufnr
```

Comment:
Would you be fine with making just `LuaSnip://Snippets` the default? The `LuaSnip://Snippets buf 0` looks a bit weird to me :sweat_smile:

Reply:
Solved in https://github.com/L3MON4D3/LuaSnip/pull/652/commits/3497f695a34301d2dac7c3b66235074361a5e39b